sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -169,18 +171,24 @@ def _get_correlation_context(key_vault_options: Optional[AzureAppConfigurationKe
 
 
 def _buildprovider(
-        connection_string: Optional[str],
-        endpoint: Optional[str],
-        credential: Optional["TokenCredential"],
-        key_vault_options: Optional[AzureAppConfigurationKeyVaultOptions],
-        **kwargs
-    ) -> "AzureAppConfigurationProvider":
-    #pylint:disable=protected-access
+    connection_string: Optional[str],
+    endpoint: Optional[str],
+    credential: Optional["TokenCredential"],
+    key_vault_options: Optional[AzureAppConfigurationKeyVaultOptions],
+    **kwargs
+) -> "AzureAppConfigurationProvider":
+    # pylint:disable=protected-access
     provider = AzureAppConfigurationProvider()
     headers = kwargs.pop("headers", {})
-    headers["Correlation-Context"] = _get_correlation_context(key_vault_options)
+
+    if os.environ.get(RequestTracingDisabledEnvironmentVariable, default="").lower() != "true":
+        headers["Correlation-Context"] = _get_correlation_context(key_vault_options)
+
     useragent = USER_AGENT
 
+    if "retry_backoff_max" not in kwargs:
```

Comment:
retry_backoff_max is the max delay between retries. This should be 1 minute: https://github.com/Azure/AppConfiguration-DotnetProvider/blob/9a76d2a1b43d137e139e329a575eee09641e51d3/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs#LL411C1-L411C1

Reply:
Should it be `retry_total`? https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/core/azure-core#configurations